docs/CloudPermissions.md

```diff
@@ -0,0 +1,58 @@
+# Assign Required Cloud Permissions For Deploying Scalar DL
+
+This guide explains how to assign required permissions for the users for deploying Scalar DL on Azure.
+
+From the security perspective, it is better to give users limited privileges to protect the environment from unwanted activities.
+
+## Azure
+
+The following JSON is a custom role that allows users to manage Azure cloud resources for Scalar DL. These are the minimum permissions for creating a Scalar DL environment on the Azure cloud.
+
+In Azure Portal, you can create the role in the Subscriptions section. Choose your subscription and select Access control (IAM) from the menu, then click `+Add` and select `Add custom role`. 
+Once the role is created, you can assign it to a user or a group from `Add role assignment` in the `+Add` menu.
+
+Please replace your subscription ID in the `assignableScopes` array. You can give `roleName` and `description` at your convenience.
+
+```json
+{   
+ "properties": {
+        "roleName": "manual-guide-test",
+        "description": "manual-guide-test",
+        "assignableScopes": [
+            "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+        ],
+        "permissions": [
```

Comment:
Is this assuming that users will create a k8s cluster from the web console?
What if users us the command line tools to create k8s cluster? It is still valid?
We don't specify how to create a cluster so if this permission doc assumes the way, it is a problem.

Reply:
> What if users us the command line tools to create k8s cluster? It is still valid?

Yes, Users can use command line tools also. Added a sentence for it in 6239a04